src/pages/Main/index.tsx

```diff
@@ -1,18 +1,147 @@
-import React from 'react';
-import Header from 'components/Header';
+/* eslint-disable camelcase */
+import React, { useState, useEffect } from 'react';
 import CanvasComponent from 'components/Canvas/Canvas';
-import { Box } from '@mui/material';
-
-const MainPage = () => <>
-    <Header />
-    <Box sx={{
-        height: '100vh',
-        display: 'flex',
-        justifyContent: 'center',
-        alignItems: 'center',
-    }}>
-        <CanvasComponent />
-    </Box>
-</>;
+import {
+    Box, Button, Grid, Typography,
+} from '@mui/material';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import PauseIcon from '@mui/icons-material/Pause';
+import ReplayIcon from '@mui/icons-material/Replay';
+import { useAppSelector } from 'hooks/redux';
+
+const MainPage = () => {
+    const [isStarted, setStarted] = useState<boolean>(() => false);
```

Comment:
В `useState` передается начальное значение самого стейта. Сейчас в стейте лежат функции: а должны - `boolean`

```suggestion
    const [isStarted, setStarted] = useState<boolean>(false);
```